### ISSUE TITLE
New Wiki and a question on Particle Accelerators

### ISSUE THREAD

**Issue by xenoflot** [turbodiesel4598/NuclearCraft#178]
Hi Turbodiesel,

I've spent a few hours creating a new wiki for NuclearCraft over at https://ftb.gamepedia.com/NuclearCraft  I chose this wiki since the other one is terribly out of date and I wanted a fresh start. The item/block icons don't work at the moment as we're waiting on some technical issues to be resolved. I'd like to clarify the use of a few blocks, please. 

There exist a Superconducting Electromagnet, and a Superconducting Supercooler. Am I correct that these are for a Particle Accelerator? Is that still a WIP or can it be built? I can't see an Accelerator Core so I'm assuming it's a WIP. Do you have any documentation on particle accelerators yet?

What's the purpose of coolers on a Fusion reactor? I created a radius-9 Fusion Reactor and simply didn't need any coolers. I simply set up a Comparator and it autoregulated at around 93%. I was only fusing Hydrogen and Lithium-7. Do other reactions generate more heat? Is there a debugging tool to boost the temperature of a fusion reactor? I set up a reactor in creative to test different fuels but it's going to take a few hours to get up to speed!

(edit: Nine hours later... my Boron-11/Boron-11 reactor was at about 94% efficiency and heat nudging the red. Again, simply using a comparator brought it into a very stable mode. So still not sure of the value of cooling a Fusion reactor!)

Do passive coolers work on a Fusion reactor? Do Superconducting Supercoolers work on a Fusion reactor?

While I'm here, a handful of suggestions...
* Add the energy requirement for base operations of machines to the JEI GUI. E.g. the Alloy Furnace uses 8000RF per operation.
* Add the effects of the Speed Upgrade E.g. Increases speed by X% for a Y% increase in power.
* Add buffer sizes to GUIs so that users can see things like the Alloy Furnace buffer currently holds 8300RF of 32000RF max.
* Add a power upgrade that increases the energy transfer rate and buffer size to compensate for the speed upgrades.
* Perhaps add the word Isotope to the isotopes to help with JEI searching. 
* Rename RadAway to AntiRad to avoid copyright problems :P
* I watched your 2.4 video when you mentioned the 'corrosive' effects of plasma. As you've implemented it,  plasma is very cool but... plasma doesn't work that way :) Perhaps change it to create a TNT or super-creeper-sized explosion at the electromagnet that breaks? I think it's fair to break some of the electromagnets. Once one explosion has happened, all the pressure from the ring is gone, so there shouldn't be more explosions, thus preventing the loss of the entire reactor. The plasma effect you've created would probably go very well for molten-salt reactor meltdowns!

**Comment by turbodiesel4598** [comment]
Hello xenoflot!

First of all, thank you so much for your work on the new wiki page. We were certainly having some problems with the last one (huge swathes of text being translated permanently into Russian, for example :P), so it's nice to have a fresh start. I have to say I was pretty blown away by the amount of info already there when I first looked through it a few days ago. I'll put out a video about it and other bits and bobs soon, and I'll switch up the links to send players to this new one instead of the old one :)
> There exist a Superconducting Electromagnet, and a Superconducting Supercooler. Am I correct that these are for a Particle Accelerator? Is that still a WIP or can it be built? I can't see an Accelerator Core so I'm assuming it's a WIP. Do you have any documentation on particle accelerators yet?

Yes, those will be for the accelerator - is it currently not available for 1.10.2 and above, but will be... one day... :P
The only info about the accelerator ever was in a YT video I made a while back, but even then the accelerator only had placeholder mechanics. As of yet, I only have vast clouds of ideas about how it will work, which will hopefully condensate as I near the time to begin implementing it.
> What's the purpose of coolers on a Fusion reactor? I created a radius-9 Fusion Reactor and simply didn't need any coolers. I simply set up a Comparator and it autoregulated at around 93%. I was only fusing Hydrogen and Lithium-7. Do other reactions generate more heat?

The active coolers are the only ones that can be used to cool fusion reactors - the passive and superconducting coolers will not do anything. They need to be placed on the diagonals of the fusion ring. They are not required, no, but if you want to squeeze that last 6 or 7 percent of efficiency out of your fusion fuels then it is possible using a certain number of active coolers. It's really only meant to be a mechanic for the super-keen players who don't mind doing a bit of maths and experimenting to find out exactly what rate of fluid supply is required to absolutely maximise the efficiency.
> Is there a debugging tool to boost the temperature of a fusion reactor? I set up a reactor in creative to test different fuels but it's going to take a few hours to get up to speed!

Such a thing does not exist explicitly, but raising the heat generation variable in the configs to the max value of 255 will massively increase the rate at which the temperature goes up :)

Now, on to your suggestions...
> Add the energy requirement for base operations of machines to the JEI GUI. E.g. the Alloy Furnace uses 8000RF per operation.

Will have to work out how to do this, but would the power consumption rate in the GUI energy tooltip be good enough? I would argue that the rate is more important than the absolute amount of energy per operation in most cases.
> Add the effects of the Speed Upgrade E.g. Increases speed by X% for a Y% increase in power.

I guess you mean in the GUI? Sure, sounds good :)
> Add buffer sizes to GUIs so that users can see things like the Alloy Furnace buffer currently holds 8300RF of 32000RF max.

Again, pretty simple.
> Perhaps add the word Isotope to the isotopes to help with JEI searching.

How about a little tooltip saying 'Isotope of ___'?
> Rename RadAway to AntiRad to avoid copyright problems :P

I actually don't think I should have much of a problem here, as I found there exists an entire company called Radaway that does shower-related stuff :P
> As you've implemented it, plasma is very cool but... plasma doesn't work that way :) Perhaps change it to create a TNT or super-creeper-sized explosion at the electromagnet that breaks? I think it's fair to break some of the electromagnets. Once one explosion has happened, all the pressure from the ring is gone, so there shouldn't be more explosions, thus preventing the loss of the entire reactor. The plasma effect you've created would probably go very well for molten-salt reactor meltdowns!

There is now a chance that unpowered e-magnets adjacent to plasma will explode instead of simply 'corroding' away, and the plasma effect has been updated to be less laggy :)

**Comment by turbodiesel4598** [comment]
Ok, will close this as all of the suggestions have been implemented, plus or minus a few differences :)
